experiments/cubic_sparse_logreg/runme.jl

```diff
@@ -66,61 +66,74 @@ function run_cubic_logreg_data(
     f = Cubic(Q, q, lam)
     g = ProximalCore.Zero()
 
+
+    x0 = zeros(n)
+    x_pert = x0 + randn(size(x0))
+
+    grad_x, _ = ProximalCore.gradient(f, x0)
+    grad_x_pert, _ = ProximalCore.gradient(f, x_pert)
+    gam_init = norm(x0 - x_pert)^2 / dot(grad_x - grad_x_pert, x0 - x_pert) 
+
+    println(gam_init)
```

Comment:
we keep this here?

Reply:
I guess, we decided to initialize like this. It was better than just randomly setting gamma = 1

Reply:
I mean the `println`

Reply:
updated it